projects/plugins/jetpack/tests/e2e/package.json

```diff
@@ -14,48 +14,39 @@
 	"author": "Automattic",
 	"scripts": {
 		"clean": "rm -rf output",
-		"distclean": "rm -rf node_modules && pnpm run clean",
-		"env-start": "./bin/env.sh start",
-		"env-stop": "./bin/env.sh stop",
-		"env-reset": "./bin/env.sh reset",
-		"tunnel-on": "pm2 start bin/ecosystem.config.js && pm2 logs --nostream --lines 4",
+		"distclean": "rm -rf node_modules",
+		"env-start": "./node_modules/jetpack-e2e-commons/bin/env.sh start",
```

Comment:
Would it be worth adding [a `bin` section](https://docs.npmjs.com/cli/v7/configuring-npm/package-json#bin) to `tools/e2e-commons/package.json` so these don't have to look inside `node_modules` to try to run things?

Reply:
Yeah, it makes sense. The plan is to put all those scripts under an `e2e cli` and that would remove all the references to `node_modules`. But until then, having a bin section will make things cleaner and I did so with [207ed57](https://github.com/Automattic/jetpack/pull/21115/commits/207ed57efcf06fa099cd8d26373045ba0c1dd844).

It doesn't seem to work for `app_password` script, which needs to be sourced in `test-e2e` command. The script exports an env variable needed by the next command. Unless I use the full path to the script it gives an error.

So far I couldn't find a nice solution for the config files.